draft-ietf-tls-rfc8446bis.md

```diff
@@ -3517,6 +3517,16 @@ a KeyUpdate with the old key is received before accepting any messages
 encrypted with the new key. Failure to do so may allow message truncation
 attacks.
 
+With a 128-bit key as in AES-128, rekeying 2^64 times has a high
+probability of key reuse within a given connection.  Note that even
+if the key repeats, the IV is also independently generated.  In order
+to provide an extra margin of security, sending implementations MUST
+NOT allow the epoch to exceed 2^48-1.  In order to allow this value
+to be changed later, receiving implementations MUST NOT enforce this
+rule.  If a sending implementation receives a KeyUpdate with
+request_update set to "update_requested", it MUST NOT send its own
+KeyUpdate if that would cause it to exceed these limits and SHOULD
+instead ignore the "update_requested" flag. 
```

Comment:
Do you need to advise that the connection be abandoned at this point, or maybe when this last set of keys expires?

Reply:
I don't think we need to. 9147 doesn't say anything about this.